packages/fast-components-react-base/src/toggle/README.md

```diff
@@ -1,11 +1,5 @@
 # Toggle
 *Toggle* mimics a physical switch that allows users to turn things **on** or **off**.
 
-## Best practices
-Use *toggle* for binary operations that become effective immediately after the user changes it. For example, use *toggle* to turn services or hardware components **on** or **off**.
-
-## Usage guidance
-Only replace the **on** and **off** labels if there are more specific labels for the setting. If there are short, 3-4 character labels that represent binary opposites that are more appropriate for a particular setting, use them. For example, you might use **show** or **hide** if the setting is "Show images." Using more specific labels can help when dealing with different languages.
-
-## Behavioral guidance
-If any extra step is required for changes to be effective, you should use *checkbox* and corresponding "Apply" button instead of *toggle*. Users shouldn’t do something else or go somewhere else in order to experience the *toggle's* effect.
\ No newline at end of file
+## Accessibility
+The functionality of *toggle* is very similar to a typical [checkbox](https://www.w3.org/TR/wai-aria-practices/#checkbox), but without the third state (partially checked). If any extra step is required for changes to be effective, you should use [checkbox](/components/checkbox/) and corresponding "Apply" button instead of *toggle*. Users shouldn’t do something else or go somewhere else in order to experience the *toggle's* effect.
```

Comment:
Is this actually accessibility guidance? It seems more generic than a11y related.